demo-app/src/main/java/org/openmined/syft/demo/federated/domain/TrainingTask.kt

```diff
@@ -39,7 +36,7 @@ class TrainingTask(
                 clientConfig: ClientConfig
             ) {
                 logger.postLog("Model ${model.modelName} received.\n\nStarting training process")
-                trainingProcess(mnistJob, model, plans, clientConfig, logger)
+                mnistJob.train(plans, mnistDataRepository, clientConfig, logger)
```

Comment:
We would not need to call train inside the training task. The user will either implement the onReady handler or call scikit learn style train method. Train method cannot subsume all types of training needs and hence it's an optional call foe the user where library implements all the basic iterators etc